Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM folioci/alpine-jre-openjdk8:latest
+FROM folioci/alpine-jre-openjdk11:latest
```

Comment:
Is there docker/docker-entrypoint.sh file in the repo? If yes, then we should remove it as well.

Reply:
Removed